smart-contracts/contracts/test-artifacts/TestProxyAdmin.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+contract TestProxyAdmin is ProxyAdmin {}
```

Comment:
This one just exist because I cant manage to import directly contract from `node_modules` into hardhat tests